.editorconfig

```diff
@@ -0,0 +1,5 @@
+root = true
```

Comment:
Is there a good reason ? 

Just wondering, 'cause I find the `.editorconfig` quite useful.